Appserver/FormSubmit/FormToDbUtil.cs

```diff
@@ -225,7 +169,7 @@ public void PopulateTimesheetEntries(TimesheetForm tsf, Timesheet tsheet)
                 tl.Add(x);
             }
 
-            tsheet.TotalHours = totalHours;
+            tsheet.TotalHours = Math.Round(totalHours, 2);

```

Comment:
For the totalHours field calculation, that value is now validated and sent as part of the PWA response, so we don't need to recalculate it in the PopulateTimesheetEntries(). See the changes in my PR.

Reply:
We'll take care of that with your patch, but since it is currently in there it isn't a problem it gets handled then removed.